README.md

```diff
@@ -57,6 +49,11 @@ backend of users' choice.
 
 Currently OpenCensus Service consists of two components,
 [OpenCensus Agent](#opencensus-agent) and [OpenCensus Collector](#opencensus-collector).
+High-level workflow:
+
+![service-architecture](https://user-images.githubusercontent.com/10536136/46707027-dae59700-cbec-11e8-80ae-f16a8bdbacf4.png)
```

Comment:
@songy23 minor nit, this is the one I was talking about :) https://user-images.githubusercontent.com/10536136/46637070-65f05f80-cb0f-11e8-96e6-bc56468486b3.png

Reply:
SG, copied that one here.

Reply:
Thanks! LGTM, but also please rebase with master.